dependencies.yaml

```diff
@@ -407,7 +407,7 @@ dependencies:
       match: "IMAGE_VERSION: 'bullseye-v((([0-9]+)\\.([0-9]+)\\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?)(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?)'"
 
   - name: "k8s.gcr.io/build-image/distroless-iptables"
```

Comment:
this class of entry with a single source file is busy work.

Reply:
also... v0.2.0 isn't even a dependency version, it's a version this repo is _producing_, right?

Reply:
right.